samples/drivers/flash/performance/sample.yaml

```
@@ -0,0 +1,12 @@
+sample:
+  description: Flash performance test
+  name: Flash Performance Test
+common:
+  tags: smoke
+  integration_platforms:
+    - tt_blackhole@p100/tt_blackhole/smc
+  harness: console
+  harness_config:
+    type: one_line
+    regex:
+      - "Flash performance test complete"
```

Maybe something like this?
```suggestion
      - "Flash performance test complete"
tests:
  sample.drivers.flash.performance: {}
```

---

It does seem like we need that- added.